packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -963,7 +968,21 @@ class ScrollAction extends Action<ScrollIntent> {
   @override
   bool isEnabled(ScrollIntent intent) {
     final FocusNode? focus = primaryFocus;
-    return focus != null && focus.context != null && Scrollable.of(focus.context!) != null;
+    final bool contextIsValid = focus != null && focus.context != null;
```

Comment:
You could just move this into the if, since you only use it once.

Reply:
Or, you could leave it.  The name does provide inherent documentation. Your choice.